### PR TITLE
#21879 editing & creation of addresses for organisational units

### PR DESCRIPTION
**Pull request by danchr**

### CODE REVIEW

File: mora/service/orgunit.py [hunk 1]
```diff
@@ -424,119 +506,157 @@ def edit_org_unit(unitid):
...
+          }
+        }
+      ]
+
+    """
+
+    reqs = flask.request.get_json()
+
+    if not isinstance(reqs, list):
+        return flask.jsonify('Root object must be a list!'), 400
```
Comment (cmoesgaard) [inline mora/service/orgunit.py]: Why not do the same thing here, of wrapping the payload in a list, if it isn't?

File: mora/service/orgunit.py [hunk 2]
```diff
@@ -424,119 +506,157 @@ def edit_org_unit(unitid):
 
     :statuscode 200: The edit succeeded.
 
-    **Example Request**:
-
     :param unitid: The UUID of the organisational unit.
```
Comment (cmoesgaard) [inline mora/service/orgunit.py]: Update the docstring of this function to make it clear that we are now editing the details for an org unit, and no longer just the org unit itself.

File: mora/service/orgunit.py (hunk 1 above)
Comment (danchr) [reply mora/service/orgunit.py]: YAGNI — wrapping is for compatibility, and this function is new?

File: mora/service/orgunit.py (hunk 2 above)
Comment (danchr) [reply mora/service/orgunit.py]: Indeed, and I'll fix the other docstring as well while at it!

File: mora/service/orgunit.py (hunk 1 above)
Comment (cmoesgaard) [reply mora/service/orgunit.py]: I was merely thinking in terms of keeping our API consistent.
If the other instance of wrapping in a list is strictly an undocumented compatiblity thing, then let's just keep this as it is.

**Pull request merged**